octree/include/pcl/octree/octree_iterator.h

```diff
@@ -448,79 +449,79 @@ namespace pcl
      * \ingroup octree
      * \author Julius Kammerl (julius@kammerl.de)
      */
-    template<typename OctreeT>
-      class OctreeBreadthFirstIterator : public OctreeIteratorBase<OctreeT>
-      {
-      public:
-        // public typedefs
-        typedef typename OctreeIteratorBase<OctreeT>::BranchNode BranchNode;
-        typedef typename OctreeIteratorBase<OctreeT>::LeafNode LeafNode;
+      template<typename OctreeT>
```

Comment:
We have a policy that we do not reformat existing code (unless we make other meaningful edits to it), the reason being that `git blame` won't show relevant commit anymore.

So please remove this restyling of the existing code (`OctreeBreadthFirstIterator`).

Reply:
I understand why existing code should not be touch. In this case, I do a tiny edit: I put the keyword `public:` just above the `typedef` lines. Should I still remove my restyling? (For the `git blame` reason, I'll understand if you say "yes"...)

Reply:
I removed the restyle of this class.

Reply:
Adding `public:` is a needed, meaningful change, we should keep it. It does not affect `git blame`, because `git blame` reports the author on line-by-line basis. If in future we have problems with this line, `git blame` will tell us that you are the one to blame :laughing:

Now as for the style changes, for example indentation fixes, if in future we discover that a certain line does something strange and we try to understand where it comes from and who are the author, `git blame` will again point at you. But this won't give us anything, because your change was only about the indentation.

Reply:
I rewrite the history for the `git blame` matter. If you do this command:
```
git blame octree/include/pcl/octree/octree_iterator.h
```
you'll see that I touch only the `public:` keyword line in the class `OctreeBreadthFirstIterator`.

Reply:
Right, if you just run `git blame` on a file you get only your last edit. What I meant was rather the feature offered by GitHub, e.g. [here](https://github.com/frozar/pcl/blame/5b3cf3f246f5bce26bdda3265dfa2544bb57b707/octree/include/pcl/octree/octree_iterator.h). You can see the last commit for each and every line. So if you scroll down you will easily see that `public:` is added by you, but other lines come from Julius.

Reply:
I didn't know this tool. It's pretty cool, thank you.